base/lttagging.dtx

```diff
@@ -621,7 +644,215 @@
 %    \end{macrocode}
 % \end{socketdecl}
 %
+% \subsection{Tagging support for math}
+%
+% \subsubsection{General sockets}
+%
+% The following sockets are the main math sockets. They are declared
+% in the math module and only listed here for reference
```

Comment:
shouldn't they be declared here instead?

If they are used in any package then not loading the math module would make that code fail

Reply:
I moved them.